notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -534,7 +503,7 @@
       "source": [
         "### Artifact\n",
         "\n",
-        "An artifact is a resource generated by your pipeline. Artifacts could datasets, models, endpoints, or custom resources defined in your pipeline."
+        "An artifact is a resource generated by your pipeline. It can be datasets, models, endpoints, or custom resources defined in your pipeline."
```

Comment:
- change: "It can be datasets, models, endpoints, or custom resources defined in your pipeline."
- to: "Artifacts can be datasets, models, endpoints, or custom resources defined in your pipeline."